apps/user-service/src/main/resources/application.yaml

```
@@ -23,13 +15,24 @@ spring:
+    config:
+      enabled: false
+      import-check:
+        enabled: false
+
+jwt:
+  secret: ${JWT_SECRET}
+  access-token-expiration: ${JWT_ACCESS_TOKEN_EXPIRATION:3600000}
+  refresh-token-expiration: ${JWT_REFRESH_TOKEN_EXPIRATION:1209600000}
+
+
 
```

YAMLlint (1.38.0)

[error] too many blank lines (3 > 2)

(empty-lines)